Docker/README.md

```diff
@@ -0,0 +1,278 @@
+# Docker 101 #
+
+## What is Docker? ##
```

Comment:
Seen someone else do this on our team. So very small nitpick but you don't need the double `#` hash marks on both sides? Only one side is needed and is considered proper markdown

```
# Header
```
vs
```
# Header #
```

Official markdown syntax: http://daringfireball.net/projects/markdown/syntax#header
Github official markdown syntax: https://guides.github.com/features/mastering-markdown/#syntax